hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/utils/TestFunctionWrapper.java

```diff
@@ -82,6 +82,14 @@ default void coordinatorFails() throws Exception {
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * Triggers Job level fail, so the coordinator need re-create a new instance.
+   * @throws Exception
+   */
+  default void coordinatorFailsAndRenewOne() throws Exception {
+    throw new UnsupportedOperationException();
+  }
+
```

Comment:
Triggers Job level fail. During the process of simulating Job restart, we need to reinitialize a new coordinator, so that the coordinator's instant will be initialized to BOOTSTRAP_INSTANT. Otherwise, the old coordinator caches the old instant, which will affect the test results.